TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithExecuteImmediateStatement.rdzSQL.txt

```diff
@@ -0,0 +1,23 @@
+﻿--- Diagnostics ---
+Line 7[14,20] <27, Error, Syntax> - Syntax error : An indicator variable must not be specified with a host variable in EXECUTE IMMEDIATE statement RuleStack=codeElement>executeImmediateStatement,  OffendingSymbol=[14,20:EXECUTE]<SQL_EXECUTE>
+Line 9[14,20] <27, Error, Syntax> - Syntax error : The constant must be Alphanumeric in EXECUTE IMMEDIATE statement RuleStack=codeElement>executeImmediateStatement,  OffendingSymbol=[14,20:EXECUTE]<SQL_EXECUTE>
+Line 10[14,20] <27, Error, Syntax> - Syntax error : Only 2 types of expression are allowed in EXECUTE IMMEDIATE statement: variable or constant RuleStack=codeElement>executeImmediateStatement,  OffendingSymbol=[14,20:EXECUTE]<SQL_EXECUTE>
```

Comment:
The message is unhelpful for the user. 'variable or constant' means anything is allowed... On line 10, we already have a syntax error coming from ANTLR, what would you like to add ?